test/test_docker_policy_run_args.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Please fix shellcheck errors.

Reply:
Ignoring 
> SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

because it requires installing mycmd, an extra dependency.